### PR TITLE
added quotes, french authors

### PR DESCRIPTION
**Pull request by Vlad412**
Hello,
Added two quotes from french authors, Ferdinand de Saussure and Valery Giscard D'Estaing.
Hope it helps 🤷‍♂️
Have a nice day 🙏

## ✅ New Quote Checklist
- [ x] The quote(s) added are in alphabetical order and not ❌ duplicates.
- [x ] The quote(s) added are about **learning a foreign language**.
- [x ] The sentence attribute does not have inner quotation marks, and is ended with a period.
- [x ] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

### CODE REVIEW

File: server/data/quotes.json [hunk 1]
```diff
@@ -155,6 +155,14 @@
     "author": "Ferdinand de Saussure",
     "sentence": "Language is a system of signs expressive of ideas."
   },
+  {
+    "author": "Ferdinand de Saussure",
+    "sentence": "Language is a treasure deposited by the practice of speech in subjects belonging to the same community."
+  },
+  {
+    "author": "Valery Giscard D'Estaing",
```
Comment (Kvaibhav01) [inline server/data/quotes.json:163]: Add this quote alphabetically towards the "V" letter quotes.

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
Thanks for the PR @Vlad412. Kindly change it as requested in the comment.

**Comment by Kvaibhav01** [comment]
Thanks for the change!

**Comment by Vlad412** [comment]
here you go 🙏
thanks and have a nice day

**Comment by jayehernandez** [comment]
Thank you for contributing to Letra ☀️, @Vlad412!! Appreciate your contribution to this open source project. 🥳

---

Continue supporting Letra by:
- Leaving a star ⭐️ for this GitHub repository
- Downloading the free [Chrome extension](https://chrome.google.com/webstore/detail/letra/cjodkkjokggcaeacdhjliobekbnnmoio) and leaving a review
- Upvoting the [ProductHunt page](https://www.producthunt.com/posts/letra-2)

**Pull request merged**